CHANGELOG.md

```diff
@@ -71,6 +71,7 @@
 * Python: Add ZSCAN and HSCAN commands ([#1732](https://github.com/aws/glide-for-redis/pull/1732))
 * Python: Added FCALL_RO command ([#1721](https://github.com/aws/glide-for-redis/pull/1721))
 * Python: Added WATCH and UNWATCH command ([#1736](https://github.com/aws/glide-for-redis/pull/1736))
+* Java: Added SCAN command ([#1751](https://github.com/aws/glide-for-redis/pull/1751))
```

Comment:
Not needed I think

Reply:
I think we should start to add it... since Java is technically "released"